### PR TITLE
Chatbot

### PR DESCRIPTION
**Pull request by payal-98**
I made a chatbot using Twilio , Python and dialogflow in which I am sending the message to whatsapp through my command line interface and receiving the response from bot,

check images in images folder

To run this set the os  environment  variables.

### CODE REVIEW

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5236736&bid=17165004)

File: whatsapp_chatbot/chatbot.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    except InvalidArgument:
+        raise
+    
+    print("Query text:", response.query_result.query_text)
+    print("Detected intent:", response.query_result.intent.display_name)
+    print("Detected intent confidence:", response.query_result.intent_detection_confidence)
+    print("Fulfillment text:", response.query_result.fulfillment_text)
+    
+    client.messages.create(
+         body=response.query_result.query_text,
```
Comment (rpotter12) [inline whatsapp_chatbot/chatbot.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5236736)

File: whatsapp_chatbot/chatbot.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
+
+import os
```
Comment (rpotter12) [inline whatsapp_chatbot/chatbot.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5236736)

File: whatsapp_chatbot/chatbot.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    client.messages.create(
+         body=response.query_result.query_text,  
+         from_=from_whatsapp_number,
+         to=to_whatsapp_number)
+    
+    client.messages.create(
+         body=response.query_result.fulfillment_text,  
+         from_=from_whatsapp_number,
+         to=to_whatsapp_number)
+
```
Comment (rpotter12) [inline whatsapp_chatbot/chatbot.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5236736)

File: whatsapp_chatbot/chatbot.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    print("Detected intent confidence:", response.query_result.intent_detection_confidence)
+    print("Fulfillment text:", response.query_result.fulfillment_text)
+    
+    client.messages.create(
+         body=response.query_result.query_text,  
+         from_=from_whatsapp_number,
+         to=to_whatsapp_number)
+    
+    client.messages.create(
+         body=response.query_result.fulfillment_text,
```
Comment (rpotter12) [inline whatsapp_chatbot/chatbot.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5236736)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5236736&bid=17165004)

File: whatsapp_chatbot/chatbot.py [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    try:
+        response = session_client.detect_intent(session=session, query_input=query_input)
+    except InvalidArgument:
+        raise
+    print("Query text:", response.query_result.query_text)
+    print("Detected intent:", response.query_result.intent.display_name)
+    print("Detected intent confidence:", response.query_result.intent_detection_confidence)
+    print("Fulfillment text:", response.query_result.fulfillment_text)
+    client.messages.create(
+         body=response.query_result.query_text,
```
Comment (rpotter12) [inline whatsapp_chatbot/chatbot.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5236736)

File: whatsapp_chatbot/chatbot.py [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    print("Query text:", response.query_result.query_text)
+    print("Detected intent:", response.query_result.intent.display_name)
+    print("Detected intent confidence:", response.query_result.intent_detection_confidence)
+    print("Fulfillment text:", response.query_result.fulfillment_text)
+    client.messages.create(
+         body=response.query_result.query_text,  
+         from_=from_whatsapp_number,
+         to=to_whatsapp_number)
+    client.messages.create(
+         body=response.query_result.fulfillment_text,
```
Comment (rpotter12) [inline whatsapp_chatbot/chatbot.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5236736)

File: whatsapp_chatbot/chatbot.py (hunk 5 above)
Comment (payal-98) [reply whatsapp_chatbot/chatbot.py]: I remove the trailing whitespaces..But I am not able to make pull request again

Comment (priyansh19) [reply whatsapp_chatbot/chatbot.py]: you don't have to make a new PR for that. When you update the changes in your repo your PR pickups all the changes itself :)

Comment (payal-98) [reply whatsapp_chatbot/chatbot.py]: ohh Thanks..can u tell mw wht is the problem?..Actually I am not able to get it

Comment (payal-98) [reply whatsapp_chatbot/chatbot.py]: Please look into this again, I think its resolved now....I understood the error

**Comment by priyansh19** [comment]
@anyone working on this PR 
No assignees @rpotter12 @xandao6

**Pull request closed without merging**